benchmarks/build.xml

```diff
@@ -3,7 +3,24 @@
 <project basedir="." default="run" name="benchmarks">
 	<import file="../build-common.xml" />
 
-	<target name="build-sample-sql">
+	<property name="sample-data-sql" value="sample-data-${sample.sql.db.type}.sql" />
```

Comment:
@slnn 
There is no "sample-data-${sample.sql.db.type}.sql" in https://github.com/liferay/liferay-portal/tree/master/benchmarks. It is confusing you have this property setting here.

I guess you need this for our benchmark project, but this is not a good reason to put it here.

Tina.

Reply:
@slnn Sorry, I misunderstood you. This is not for property reading, it is for output, right?